inductiva/design.py

```diff
@@ -30,13 +30,14 @@ def explore_design_space(simulator: Simulator,
         extra_sim_kwargs: Extra keyword arguments to pass to the simulator.
     """
     input_dir = pathlib.Path(input_dir)
+    file_format = os.path.splitext(template_filename)[1]
```

Comment:
Since `pathlib` is being used here, I think it's better to use it instead of `os.path`. `pathlib.Path(template_filename).suffix` can be used to extract the extension from the path